### DISCUSSION TITLE
How to use OWSM model? Getting `TypeError` in `Speech2Text.from_pretrained()`

### DISCUSSION THREAD

**Discussion by cifkao** [espnet/espnet#5723] [Q&A] [answered]
I'm excited to try the OWSM 3.1 model, but the snippet provided [on HuggingFace](https://huggingface.co/espnet/owsm_v3.1_ebf) under "Use in ESPnet" does not work for me:
```python
from espnet2.bin.asr_inference import Speech2Text

model = Speech2Text.from_pretrained(
  "espnet/owsm_v3.1_ebf"
)
```
I get the following error:
```pytb
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[1], line 3
      1 from espnet2.bin.asr_inference import Speech2Text
----> 3 model = Speech2Text.from_pretrained(
      4   "espnet/owsm_v3.1_ebf"
      5 )

File ~/mambaforge/envs/espnet/lib/python3.10/site-packages/espnet2/bin/asr_inference.py:679, in Speech2Text.from_pretrained(model_tag, **kwargs)
    676     d = ModelDownloader()
    677     kwargs.update(**d.download_and_unpack(model_tag))
--> 679 return Speech2Text(**kwargs)

TypeError: Speech2Text.__init__() got an unexpected keyword argument 's2t_train_config'
```
I tried `espnet/owsm_v3` too and it doesn't work either.

What is the correct code to load the model and run transcription? Thanks!

**Comment by sw005320** [comment, marked as answer]
It should be `espnet2.bin.s2t_inference` not `espnet2.bin.asr_inference`

@pyf98, we encountered this misuse in two cases.
- Is there a way to avoid this issue? (at least provide some more meaningful error messages?)
- We have to update "Use in ESPnet" in 
<img width="1464" alt="image" src="https://github.com/espnet/espnet/assets/11741550/c64a3a0a-76d2-460d-8777-42ce3d943656">
This actually shows `espnet2.bin.asr_inference`
<img width="968" alt="image" src="https://github.com/espnet/espnet/assets/11741550/12cf7561-9d17-4dc8-ad3e-09d91a3327cd">

**Comment by pyf98** [comment]
yeah, I see. We need to update "Use in ESPnet". I think it is defined in Hugging Face not ESPnet?
